pkg/proc/target_group.go

```diff
@@ -46,6 +55,9 @@ func NewGroup(t *Target) *TargetGroup {
 		LogicalBreakpoints:    t.Breakpoints().Logical,
 		continueOnce:          t.continueOnce,
 	}
+	grp.cctx.grp = grp
+	grp.cctx.DebugInfoDirs = t.debugInfoDirs
```

Comment:
Should `debugInfoDirs` just live on `TargetGroup` and then let `Target` have a reference back up to the group?

Reply:
The way initialization is done right now, we make a target with NewTarget, which creates a BinaryInfo (and therefore needs debugInfoDirs) and then we create a group with NewTargetGroup(target). So the target group doesn't exist when we need debugInfoDirs (at least, the first time). To do this we would have to make Launch, Attach etc return a TargetGroup.

Reply:
I'm not against making Launch, Attach, etc return a TargetGroup since that's the level of abstraction we deal with now. Thoughts?